src/services/monaco-editor.service.ts

```diff
@@ -81,6 +81,15 @@ export class MonacoEditorService {
 			});
 	}
 
+  /**
```

Comment:
Can you fix the indentation? There's a `.editorconfig` but maybe it failed to pick it up.

Reply:
Yupp!